src/components/dev-hub/stories/hero-banner.stories.mdx

```diff
@@ -10,18 +10,23 @@ import MockAuthorImage from '../../../images/1x/MDB-and-Node.js.png';
 
 <Preview>
     <Story name="hero banner">
-        <HeroBanner
-            background={MockAuthorImage}
-            breadcrumb={[
+        {() => {
```

Comment:
Why do we need an inner function now?

Reply:
No we shouldn't need it. I originally made this change while trying out some updates to resolve issues with `prettier`.